README.md

```diff
@@ -447,3 +448,11 @@ Finally, remove the ec2 instance.
 ```sh
 ./.github/scripts/cleanup-ec2.sh
 ```
+
+## Creating Additional Images
```

Comment:
Instead of adding docs here, I would suggest centralizing them in the DANDI Docs - https://docs.dandiarchive.org/50_hub/#custom-server-image.

We should also add notes about updating the `docker-test.yaml`, `docker-push.yaml` and `jupyterhub.yaml` files.